script/nsis/installer.nsi

```diff
@@ -0,0 +1,66 @@
+Name    "Git LFS"
+OutFile "GitLFSInstaller.exe"
+
+InstallDir $DESKTOP
+
+VIProductVersion "0.6.0.0"
+VIAddVersionKey FileVersion      "0.6.0.0"
+VIAddVersionKey ProductVersion   "0.6.0.0"
+VIAddVersionKey ProductName      "Git LFS"
+VIAddVersionKey Comments         "Git LFS"
+VIAddVersionKey CompanyName      "GitHub, Inc"
+VIAddVersionKey FileDescription  "Git LFS extension"
+VIAddVersionKey InternalName     "Git LFS"
+VIAddVersionKey LegalCopyright   "(c) GitHub, Inc. and Git LFS contributors"
+VIAddVersionKey LegalTrademarks  "(c) GitHub, Inc. and Git LFS contributors"
+VIAddVersionKey OriginalFilename "GitLFSInstaller.exe"
+
+Function .onGUIInit
+  Var /GLOBAL HASPROXYGIT
+  StrCpy $HASPROXYGIT "no"
+  IfFileExists $PROGRAMFILES64\Git\cmd\git.exe 0 +2
+    StrCpy $HASPROXYGIT "yes"
+
+  ; Git for Windows location
+  IfFileExists $PROGRAMFILES64\Git\mingw64\bin\git.exe 0 +3
+    StrCpy $INSTDIR "$PROGRAMFILES64\Git\mingw64\bin"
+    Goto exit
+
+  IfFileExists $PROGRAMFILES64\Git\bin\git.exe 0 +3
+    StrCpy $INSTDIR "$PROGRAMFILES64\Git\bin"
+    Goto exit
+
+  IfFileExists $PROGRAMFILES\Git\bin\git.exe 0 +2
+    StrCpy $INSTDIR "$PROGRAMFILES\Git\bin\git.exe"
```

Comment:
You need to remove the '\git.exe', when you run this against a 32-bit install the "Install" button is disabled because it's pointing at an existing file instead of a folder.